companies/services/sankey.py

```
@@ -0,0 +1,134 @@
+        return {"status": "success"}
+
+    def sync_right_side(self, company, year=2024):
+        try:
+            fs = FinancialStatement.objects.filter(company=company, fiscal_year=year, report_code='11011').first()
+            report = Report.objects.filter(company=company, report_name__contains=str(year)).first()
+            ext_info = report.extracted_info if report else {}
+            
+            params = {'crtfc_key': self.api_key, 'corp_code': company.corp_code, 'bsns_year': str(year), 'reprt_code': '11011', 'fs_div': 'CFS'}
+            res = requests.get(self.url, params=params, timeout=self.REQUEST_TIMEOUT).json()
+            if res.get('status') != '000':
+                params['fs_div'] = 'OFS'; res = requests.get(self.url, params=params, timeout=self.REQUEST_TIMEOUT).json()
```

Ruff (0.14.13)

Multiple statements on one line (semicolon)

(E702)